redwood-gradle-plugin/src/main/kotlin/app/cash/redwood/gradle/RedwoodLintTask.kt

```diff
@@ -40,8 +37,7 @@ internal abstract class RedwoodLintTask @Inject constructor(
   @get:Classpath
   abstract val toolClasspath: ConfigurableFileCollection
 
-  @get:InputDirectory
-  @get:PathSensitive(PathSensitivity.ABSOLUTE)
+  @get:Input
```

Comment:
This doesn't need reverted, right? It was just incorrect always.

Reply:
Yep! I also removed `@get:PathSensitive(PathSensitivity.ABSOLUTE)` since `ABSOLUTE` is the default.